Python/CMakeLists.txt

```diff
@@ -1,22 +1,14 @@
 ## Python module ##
 
-option(BUILD_PYTHON_INTERFACE "Build the Python interface" ON)
+option(INSTALL_PYTHON_INTERFACE "Build the Python interface" ON)
```

Comment:
Can't really rename this as it would break the public API that people are using

Reply:
Well, I added the option in https://github.com/csound/csound/pull/1744 two months ago, so I doubt anyone is relying on it, but happy to use the original name